client/my-sites/plans-features-main/index.tsx

```diff
@@ -891,13 +874,13 @@ const PlansFeaturesMain = ( {
 									isCustomDomainAllowedOnFreePlan={ isCustomDomainAllowedOnFreePlan }
 									isInSignup={ isInSignup }
 									isLaunchPage={ isLaunchPage }
-									onUpgradeClick={ handleUpgradeClick }
 									selectedFeature={ selectedFeature }
 									selectedSiteId={ siteId }
 									intervalType={ intervalType }
 									hideUnavailableFeatures={ hideUnavailableFeatures }
 									currentSitePlanSlug={ sitePlanSlug }
 									planActionOverrides={ planActionOverrides }
```

Comment:
I think we ought to get rid of these `planActionOverrides` now, if we are centralising the logic behind `planActions`. Otherwise it feels weird to be passing custom actions and then custom overrides to those actions. Are there blockers to doing so? 🤔

Reply:
Still wrapping my head around the work here, but I'll plan to remove `planActionOverrides` for the time being :)